regression/cbmc/union11/union_list.c

```diff
@@ -0,0 +1,30 @@
+#include <assert.h>
+
+struct list_item
+{
+  struct list_item *previous;
```

Comment:
Why a single member struct instead of simply inlining this?

Reply:
We could have more members, just don't need more to demonstrate the problem/the fix.